### DISCUSSION TITLE
composer file please support php5.4

### DISCUSSION THREAD

**Discussion by zlseqx** [laravel/framework#33536] [General] [answered]
I use php5.4 to create the project, but laravel depends on doctrine/inflector: ~ 1.0, it should be: doctrine/inflector: 1.0.*?

![image](https://user-images.githubusercontent.com/26402118/87424874-591b6c80-c60f-11ea-9119-a0b012141755.png)

**Comment by driesvints** [comment, marked as answer]
PHP 5.4 is EoL and it's highly discouraged to use it. Please upgrade to a maintained version of PHP like 7.4.

**Comment by GrahamCampbell** [comment]
> I use php5.4 to create the project, but laravel depends on doctrine/inflector: ~ 1.0, it should be: doctrine/inflector: 1.0.*?

Laravel also depends on laravel/framework, which needs php 7.2+
